core/external_search.py

```diff
@@ -2052,7 +2053,9 @@ class Operators(Values):
         "identifiers.identifier": dict(path="identifiers"),
         "identifiers.type": dict(path="identifiers"),
         "imprint": _KEYWORD_ONLY,
-        "language": dict(),
+        "language": dict(
+            type="_text"
+        ),  # Made up keyword type, because we don't want text fuzzyness on this
```

Comment:
This change avoids fuzziness on the language field, since it is no longer of `type=text` (by default)